src/aihwkit/nn/low_precision_conversion.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+
+# (C) Copyright 2020, 2021, 2022, 2023, 2024 IBM. All Rights Reserved.
+#
+# Licensed under the MIT license. See LICENSE file in the project root for details.
+
+""" Functions to convert a given model to a quantized counterpart """
+
+import copy
```

Comment:
To be consistent with the other code imports, can you import deeopcopy from copy and use this directly, e.g., deepcopy(x) rather than copy.deepcopy(x)?